### DISCUSSION TITLE
Suggestion - Add RR73 and 73 lines to the listing in the WSJTX window

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#162] [Ideas]
Hi Ladislav,
Today the listing in the WSJTX window only include CQ calls. 

Many DX stations have many callers, ending up never or seldom calling CQ, the DX station answers directly to tail-ending calls after sending RR73 or 73. 

Different user categories may have different perspectives on this, but giving the user an OPTION to include RR73 and 73 in the listing, would be a great feature. 

 73 de LB2EG Richard

**Comment by foldynl** [comment]
we need to think more about this. In general, I don't want QLog to copy the WSJTX functions. That's what the WSJTX app should be used for, to display complete traffic. The main reason for implementing the window was to provide a simplified view of the traffic. Remove in-progress QSOs and show CQ-stations. I know this feature already exists in WSJTX, but it only started to make sense after adding various filters.

However, if we also display stations that send RR73 or 73, the window will also display stations that do not call CQ. That's why I don't think we should mindlessly insert RR73 spots. But I will try to look at it in more detail.
